ansible/roles/jenkins/tasks/tools.yml

```
@@ -0,0 +1,40 @@
+---
+
+- name: Upload gcp credentials
+  ansible.builtin.copy:
+    src: "whanos_gcp_creds.json"
+    dest: /var/lib/jenkins/whanos_gcp_creds.json
+    mode: preserve
+  become: true
+
+- name: Zip docker images folder
+  ansible.builtin.archive:
```

fqcn[canonical]: You should use canonical module name community.general.archive instead of ansible.builtin.archive
```suggestion
  community.general.archive:
```